drivers/dac/Kconfig.mcux

```diff
@@ -19,6 +19,13 @@ config DAC_MCUX_DAC32
 	help
 	  Enable the driver for the NXP Kinetis MCUX DAC32.
 
+config DAC_MCUX_LPDAC
+	bool "NXP Kinetis MCUX LPDAC driver"
```

Comment:
Given the discussion about naming we had before, this is probably wrong?

Reply:
I agree that we should remove "kinetis" at least from any terminology. "mcux" I think is loosely used too but comes from the name of the NXP MCU SDK which is called MCUX SDK.

Reply:
Would `CONFIG_DAC_NXP_LPDAC` not be the most self-explanatory then? MCUX seems to be used also [by other vendors](https://www.businesswire.com/news/home/20230627745063/en/Bluespec-Launches-New-MCUX-RISC-V-Processor-That-Enables-Developers-to-Implement-Custom-Instructions-and-Add-Accelerators).

Reply:
That would be an acceptable Kconfig name, yes, we don't need to have mcux in the name

Reply:
Ok. It would also match the compatible.

@Albort12138 can you please change to `CONFIG_DAC_NXP_LPDAC`? After that we should be ready to go.